server/service/handler.go

```diff
@@ -520,9 +520,20 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	// for backwards compatibility.
 	mdmAppleMW.GET("/api/_version_/fleet/mdm/apple/profiles/summary", getMDMAppleProfilesSummaryEndpoint, getMDMAppleProfilesSummaryRequest{})
 
+	// Deprecated: POST /mdm/apple/enrollment_profile is now deprecated, replaced by the
+	// POST /enrollment_profiles/automatic endpoint.
 	mdmAppleMW.POST("/api/_version_/fleet/mdm/apple/enrollment_profile", createMDMAppleSetupAssistantEndpoint, createMDMAppleSetupAssistantRequest{})
+	mdmAppleMW.POST("/api/_version_/fleet/enrollment_profiles/automatic", createMDMAppleSetupAssistantEndpoint, createMDMAppleSetupAssistantRequest{})
```

Comment:
I remember you mentioned the `WithAltPaths` method (which I thought was an awesome idea), I'm just curious why you didn't use it for my own understanding.

Was it a consistency issue with all the other paths we already renamed, or is there something about that method that's not useful?